Settings.h

```diff
@@ -110,6 +110,7 @@ typedef struct Settings_ {
    bool topologyAffinity;
    #endif
 
+   char* stateFilter;
```

Comment:
I thought the `Settings` structure is a wrong place for `stateFilter`. The `Settings` object is for things that would be saved and loaded in `htoprc` file, and the filters shouldn't carry across restarts of htop.